tests/playbooks/tests_eth_pci_address_match.yml

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: pseudo-pci-test-only
+            persistent_state: present
+            type: ethernet
+            # `pci-0001:00:00.0` is a pseudo PCI address,
+            # which used only for testing purpose
+            match:
+              path:
+                - pci-0001:00:00.0
+            ip:
+              dhcp4: no
+              address:
+                - 192.0.2.3/24
+    - block:
+        - name: Verify nmcli connection profile
+          shell: |
+            set -euxo pipefail
```

Comment:
this seems to be wrong

Reply:
should use `command` instead of `shell` here

Reply:
Correct. I changed to use `command` instead.